### DISCUSSION TITLE
Passing in command line arguments to layouts

### DISCUSSION THREAD

**Discussion by RemainCurly** [zellij-org/zellij#3350] [Q&A]
Hi! I was wondering if it was possible to use CLI args inside of layout files? For instance, if I called 
`zellij --layout myDir/zellij/my-layout.kdl`
And `my-layout.kdl` looked like: 
```kdl
layout {
	pane split_direction="vertical" {
		pane name="Logs" command="stern" args="{{ CLI_argument_would_go_here }}"
		pane size="20%" name="Pods" command="k9s" focus=true
	}
}
```
Would it be possible to do something like `zellij --layout myDir/zellij/my-layout.kdl "someApp"` so that the "Logs" pane would execute the command `stern someApp`? Thanks!

**Comment by cristiand391** [comment]
This isn't supported yet, you can follow this issue if there's future updates:
https://github.com/zellij-org/zellij/issues/2482
